agent/util-scripts/test-bin/ssh

```diff
@@ -21,16 +21,45 @@ done
 remote=${1}
 shift
 
+function mkdirp {
+    if [[ -d ${1}/${2} ]]; then
+        return 0
+    fi
+    (cd ${1}; mkdir -p ${2})
```

Comment:
This purpose of this is to use `cd` to test for the existence of `${1}` before creating `${2}`?

Shouldn't that `;` be a `&&`??

(It would be nice to have a block comment explaining the motivation for using `mkdirp` instead of `mkdir -p`....)